sunpy/instr/lyra.py

```diff
@@ -15,23 +15,35 @@
 from sunpy import config
 from sunpy.util.net import check_download_file
 from sunpy.util.config import get_and_create_download_dir
-from sunpy import lightcurve
+from sunpy.util.decorators import deprecated
+from sunpy import timeseries
```

Comment:
Nor is config, it turns out. Is that doing mysterious things that mean it needs to stay?